AppiumForMac/Server/Controller/AfMElementLocator.m

```diff
@@ -204,12 +204,12 @@ -(void)findAllUsingBaseUIElement:(PFUIElement*)baseUIElement results:(NSMutableA
 		if (error != nil)
 		{
 			*statusCode = kAfMStatusCodeXPathLookupError;
-			return;
+			return nil;
 		}
 		if (matches.count < 1)
 		{
 			*statusCode = kAfMStatusCodeNoSuchElement;
-			return;
+            return nil;
```

Comment:
Please respect the indentation

Reply:
Yeah this was weird. Cus there actually was no indentation change on my source code in xcode. So i have no idea where this indentation came from. But I'll look into it.